openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGResources.java

```diff
@@ -198,6 +213,41 @@ private void initBoosterFastPredictHandle(final String LightGBMParameters) throw
         }
     }
 
+    /**
+     * Initializes the FastConfigContributions object from a Booster.
+     * This FastConfig is responsible for caching Booster+Contributions settings for repeated prediction calls.
+     * It is used in the *Fast() predict methods instead of the Booster and prediction settings.
+     *
+     * @param LightGBMParameters String with custom LightGBM parameters.
+     * @throws ModelLoadingException in case there is a C++ back-end error creating the FastConfigContributions object.
+     * @since 1.2.2
+     */
+    private void initBoosterFastContributionsHandle(final String LightGBMParameters) throws ModelLoadingException {
```

Comment:
this method is very similar to the `initBoosterFastPredictHandle` that is used when scoring events. do you know if it is possible to use a single call to lightgbm to get both scoring and feature contribution?

Reply:
answered in the first thread